NDB.Covid19/NDB.Covid19/ViewModels/QuestionnaireViewModel.cs

```diff
@@ -59,7 +60,7 @@ public class QuestionnaireViewModel
         static DateTime _selectedDateUTC { get; set; }
         static DateTime _localSelectedDate => DateTime.SpecifyKind(_selectedDateUTC, DateTimeKind.Utc).ToLocalTime();
         public static QuestionaireSelection Selection { get; private set; } = QuestionaireSelection.Skip;
-        public DateTime MinimumDate { get; private set; } = new DateTime(2020, 11, 1, 0, 0, 0).ToUniversalTime();
+        public DateTime MinimumDate { get; private set; } = SystemTime.Now().AddDays(-21);
```

Comment:
Remove setters for minimum and maximum date. We should not be able to change it in any way.